src/main/java/bio/terra/cli/serialization/userfacing/resources/UFAiNotebook.java

```diff
@@ -59,10 +59,10 @@ public void print() {
     OUT.println("GCP project id:                " + projectId);
     OUT.println("AI Notebook instance location: " + location);
     OUT.println("AI Notebook instance id:       " + instanceId);
-    OUT.println("Instance name: " + instanceName == null ? "(undefined)" : instanceName);
-    OUT.println("State:         " + state == null ? "(undefined)" : state);
-    OUT.println("Proxy URL:     " + proxyUri == null ? "(undefined)" : proxyUri);
-    OUT.println("Create time:   " + createTime == null ? "(undefined)" : createTime);
+    OUT.println("Instance name: " + (instanceName == null ? "(undefined)" : instanceName));
```

Comment:
Optional/ofNullable/orElse works well for this situation.

Reply:
I prefer not creating objects where it's not needed. Just a style preference.